designer/settings/base.py

```diff
@@ -221,3 +221,12 @@
 LOGGING = get_logger_config(debug=DEBUG, dev_env=True, local_loglevel='DEBUG')
 
 WAGTAIL_SITE_NAME = 'Designer'
+
+CERTIFICATE_LANGUAGES= {
+    'en': 'English',
+    'es_419': 'Spanish'
+}
+DESIGNER_SERVICE_USER = 'designer_service_user'
+CSRF_COOKIE_SECURE = False
+SESSION_EXPIRE_AT_BROWSER_CLOSE = False
+SOCIAL_AUTH_REDIRECT_IS_HTTPS = False
```

Comment:
Please be sure the base edx_django_service defaults get put in here too:


You need to include all these values as well:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L182-L222

as they get merged into the edx_django_service_config here:
https://github.com/edx/configuration/blob/master/playbooks/roles/edx_django_service/defaults/main.yml#L226

Reply:
I have added all these varaiables and also I have moved databases to local.py and copy caches from local.py to base.py